OpenSim/Simulation/Model/Model.cpp

```diff
@@ -2071,7 +2096,7 @@ void Model::realizeReport(const SimTK::State& state) const
 void Model::computeStateVariableDerivatives(const SimTK::State &s) const
 {
     try {
-        realizeAcceleration(s);
+        extendRealizeAcceleration(s);
```

Comment:
I believe I mistakenly changed the method call when trying to override extendRealizeAcceleration().